src/library_webgl2.js

```diff
@@ -976,13 +976,18 @@ var LibraryWebGL2 = {
     _glDrawElements(mode, count, type, indices);
   },
 
+  glDrawArraysInstancedBaseInstanceWEBGL_sig: 'viiiii',
   glDrawArraysInstancedBaseInstanceWEBGL: function(mode, first, count, instanceCount, baseInstance) {
     GLctx.dibvbi['drawArraysInstancedBaseInstanceWEBGL'](mode, first, count, instanceCount, baseInstance);
   },
+  glDrawArraysInstancedBaseInstance: 'glDrawArraysInstancedBaseInstanceWEBGL',
+  glDrawArraysInstancedBaseInstanceANGLE: 'glDrawArraysInstancedBaseInstanceWEBGL',
 
+  glDrawElementsInstancedBaseVertexBaseInstanceWEBGL_sig: 'viiiiii',
```

Comment:
sig has 6 params, but the function seems to have 7?

Reply:
😰 Fixed. Checked the other three.